index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="icon" href="./favicon.ico">
+  <link href="./assets/css/style.css" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+
+    <a href="/" class="logo">
+      <h1 class="sr-only">판다마켓</h1>
+    </a>
+
+    <a href="/login" class="btn-sm btn-primary">로그인</a>
```

Comment:
```suggestion
    <a href="/login.html" class="btn-sm btn-primary">로그인</a>
```

`.html` 확장자를 붙여주어서 이동이 가능하게 만들어주세요 !

Reply:
> `.html` 확장자를 붙여주어서 이동이 가능하게 만들어주세요 !

넵

Reply:
> `.html` 확장자를 붙여주어서 이동이 가능하게 만들어주세요 !

넵